File: Atcoder_#351/ruby/a_the_bottom_of_the_ninth.rb
a1,a2,a3,a4,a5,a6,a7,a8,a9 = gets.split.map(&:to_i)
b1,b2,b3,b4,b5,b6,b7,b8 = gets.split.map(&:to_i)


A = a1+a2+a3+a4+a5+a6+a7+a8+a9
B = b1+b2+b3+b4+b5+b6+b7+b8

ans = A - B + 1

puts ans

File: Atcoder_#351/javascript/a_the_bottom_of_the_ninth.js
function main(input) {
  const lines = input.trim().split("\n");
  const [A1,A2,A3,A4,A5,A6,A7,A8,A9]  = lines[0].split(" ").map(Number);
  const [B1,B2,B3,B4,B5,B6,B7,B8]  = lines[1].split(" ").map(Number);

  let A = A1+A2+A3+A4+A5+A6+A7+A8+A9;
  let B = B1+B2+B3+B4+B5+B6+B7+B8;

  let ans = A - B +1

  console.log(ans);
}

main(require("fs").readFileSync("/dev/stdin", "utf8"));
